src/pyFAI/detectors/sensors.py

```diff
@@ -153,10 +153,14 @@ def absorbance(self, energy:float, length: float, unit:str="m") -> float:
 Si_MATERIAL = SensorMaterial("Si", 2.329)
 CdTe_MATERIAL = SensorMaterial("CdTe",  5.85)
 GaAs_MATERIAL = SensorMaterial("GaAs", 5.3176)
+Ge_MATERIAL = SensorMaterial("Ge", 5.327)
+Gd2O2S_MATERIAL = SensorMaterial("Gd2O2S", density=7.32)
```

Comment:
@kif Small note, "Gd2O2S" gets here explicitly the density argument. Both are valid, just a comment for consistency